codegen/test/name_override_chart/crds/things.test.io_crds.yaml

```diff
@@ -4,19 +4,19 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    crd.solo.io/specHash: 80e7931efac2214e
+    crd.solo.io/specHash: 2c0000aa4677e683
   labels:
     app: ""
     app.kubernetes.io/name: ""
-  name: paints.things.test.io
+  name: clusterresources.things.test.io
```

Comment:
Did these changes result in the ordering of CRDs change?

Reply:
Yes. There are a few interim maps: for grouping of groups, which also affected ordering of versions in CRDs. To make those stable, I added sorting in both places:
- https://github.com/solo-io/skv2/pull/462/files#diff-0edd4bd3f13b4db965a1337273d05e142d5c93e63559722098c88c9873458997R37
- https://github.com/solo-io/skv2/pull/462/files#diff-0edd4bd3f13b4db965a1337273d05e142d5c93e63559722098c88c9873458997R42